src/neo/neo.csproj

```diff
@@ -30,6 +30,7 @@
     <PackageReference Include="K4os.Compression.LZ4" Version="1.2.16" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="6.0.4" />
     <PackageReference Include="Neo.VM" Version="3.2.0-CI00312" />
+	<PackageReference Include="Neo.Cryptography.BLS12_381" Version="0.0.2" />
```

Comment:
I think that this repo must be located in `neo-project` organization.

Reply:
Now it is in NDG repo. The publishing authority of nuget is controlled by erikzhang.

Reply:
If you want to move the position of the code, just copy it directly. But I don't have permission.

Reply:
Two libraries `BLS12_381` and `BLS12_381.Native` are used here. Among them, `BLS12_381` is a wrapper and implemented in C#. `BLS12_381.Native` is a native library, implemented in rust, which involves some implementation details of the BLS curve internally. If it is ported using c#, it may be difficult and risky.

Reply:
I will port https://github.com/zkcrypto/bls12_381 to c#.